webextensions/api/webRequest.json

```
@@ -124,7 +124,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/MAX_HANDLER_BEHAVIOR_CHANGED_CALLS_PER_10_MINUTES",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "24"
```

:x: https://github.com/chromium/chromium/commit/85231d73b03552f369a32ce640189c531e42cf96 landed in [23.0.1253.0](https://github.com/chromium/chromium/releases/tag/23.0.1253.0) before 23.0.1271.

```suggestion
                "version_added": "23"
```